.github/workflows/full-test.yml

```diff
@@ -18,13 +18,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           python-version: 3.8
-      - name: Which python
-        run: |
-          which python
       - uses: actions/cache@v2
         id: cache-venv
         with:
-          path: ./test_env
+          path: ~/test_env
           key: ${{ runner.os }}-venv-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/requirements_testing.txt') }}
```

Comment:
It would be good to include some time stamp in the hash, to ensure that the environment is build from scratch at least once per week / month. Maybe it's possible to use the github actions schedule for this, too? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule